client/src/pages/ManageBookings/ManageBookings.tsx

```diff
@@ -0,0 +1,98 @@
+import DateFnsUtils from '@date-io/date-fns';
+import { Box, Grid, Paper, Typography, useTheme } from '@material-ui/core';
+import { DatePicker, MuiPickersUtilsProvider } from '@material-ui/pickers';
+import { MaterialUiPickersDate } from '@material-ui/pickers/typings/date';
+import { cloneElement, useState } from 'react';
+import { CurrentBookings, NextBooking, PastBookings } from '../../components/ManageBookings/';
+import { checkIfBooked } from '../../helpers/ManageBookings';
+import mockBookingData from './mockBookingData';
+import { BookingRequests } from './types';
+import useStyles from './useStyles';
+
+export default function ManageBookings(): JSX.Element {
```

Comment:
This component is the perfect location to move over processing logic of Bookings other than a `React.Context`!